packages/main/src/Calendar.ts

```diff
@@ -52,6 +52,12 @@ interface ICalendarPicker {
 	_lastYear?: number,
 }
 
+/**
+ * Interface for components that may be slotted inside a `ui5-calendar`.
+ *
+ * **Note:** Use with `ui5-date` or `ui5-date-range` as calendar date selection types. Implementing the interface does not guarantee that any other class can work as a calendar date selection type.
+ * @public
```

Comment:
I'm not sure about following note: **Implementing the interface does not guarantee that any other class can work as a calendar date selection type.**. The main idea of interfaces is to describe minimum required implementation to make something to work. Please discuss with the team.

Reply:
The note is similar to the one for ITab. The interface has only optional properties and internally we check for types so just implementing the interface won't be enough.

Reply:
Regarding the "Implementing the interface does not guarantee that any other class can work as a calendar date selection type." sentence.  We can remove it and change the first sentence to:
**Note:** Use only with `ui5-date` or `ui5-date-range` as calendar date selection types.
Alternatively, if we want to allow the usage of other classes we can list them. But if listing is not possible I would suggest to leave the note as it is.

Reply:
I just removed the sentence altogether as @vladitasev proposed.